### PR TITLE
Supported the correct basic list of allowed headers

### PR DESCRIPTION
**Pull request by lukasz-pomianek**
Hi @marysieek,

If you would have some time, please take a look at the updated code so that the list of allowed/denied headers could be supported.
Thanks,
LP

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 ## master
 
+## 1.3.2 (2020-08-10)
```
Comment (marysieek) [inline CHANGELOG.md:5]: Please add 1.3.1 and 1.3.2 changes under ## master

File: src/Castle.Sdk/Castle.Sdk.csproj [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;netcoreapp3.1;net48</TargetFrameworks>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <PackageProjectUrl>https://github.com/castle/castle-dotnet</PackageProjectUrl>
     <RootNamespace>Castle</RootNamespace>
-    <Version>1.3.0</Version>
+    <Version>1.3.2</Version>
```
Comment (marysieek) [inline src/Castle.Sdk/Castle.Sdk.csproj:8]: Please don't bump the version yet - we do that in separate commits

Comment (lukasz-pomianek) [reply src/Castle.Sdk/Castle.Sdk.csproj]: Reverted back to be 1.3.0

File: CHANGELOG.md (hunk 1 above)
Comment (lukasz-pomianek) [reply CHANGELOG.md]: Removed in-midst-versions, left both changes within the master branch
